docs/source/userguide/plugin_development/algorithm.rst

```diff
@@ -110,7 +110,8 @@ calculated.
 Adding Your Algorithm to GeoIPS Namespace
 -----------------------------------------
 
-Now that we've created our custom algorithm, we need to add an `entry-point <https://packaging.python.org/en/latest/specifications/entry-points/>`_
+Now that we've created our custom algorithm, we need to add an `entry-point
```

Comment:
Ooh, this is outdated,  but I think that is outside scope for this PR ...

Reply:
If it's super out of date - so much so that removing it would be better than having incorrect info - I'd like to remove it in this PR. If it is close enough, but needs updating, I think you're right and we should save it for another PR

Reply:
We no longer add entry points for every plugin!  That is handled by create_plugin_registries now, and we only have an entry point for the plugin package as a whole!!!

Reply:
Removed for now!

Reply:
I'm guessing there are other issues with this algorithm description. since the entry points were outdated I'm guessing other things were outdated also,  but I'm good with merging this as is, and the algorithm docs will be updated at a later date